package.json

```diff
@@ -2,10 +2,14 @@
   "name": "babajka-frontend",
   "version": "2.1.1",
   "description": "Next.js & React app",
+  "engines": {
+    "node": "~15",
+    "npm": "~7.1.2"
+  },
   "scripts": {
     "start": "npm run dev",
     "start:prod": "npm run build && cross-env NODE_ENV=production node server.js",
-    "start:prod:remote": "node server.js",
+    "start:prod:remote": "node -v && node server.js",
```

Comment:
Remove `node -v` ?

Reply:
If we want to print remote node version, we can put `node -v` to bash script explicitly